iotilecore/tox.ini

```diff
@@ -8,4 +8,5 @@ deps=
     ../iotiletest
     ../iotileemulate
     pycryptodome
+    iotile-support-con-nrf52832-3
```

Comment:
This dependency can't go here.  If there's been a code change that makes `iotile-core` need this installed before it works, then that code change needs to be reverted.

No open-source package can have a dependency on a closed source package.

Reply:
Ah, sorry, for testing?  My mistake.  That would be fine.

Reply:
This is a dependency on a tox test. When I ran tox on the root directory of coretools I noticed many closed-source packages in the tox dependencies.

Reply:
Here's the list of dependencies that wer einstalled when I ran tox from the root directory:
```
py36-linux_only installdeps: six, cmdln, pytest, pytest-logging, pytest-localserver, ./iotilecore, ./iotilebuild, ./iotiletest, ./transport_plugins/socket_lib, ./transport_plugins/websocket, ./iotilegateway, ./iotilesensorgraph, ./iotileship, ./iotileemulate, ./transport_plugins/bled112, ./transport_plugins/awsiot, ./transport_plugins/jlink, iotile-support-lib-
controller-4, iotile-support-con-nrf52832-3, iotile-support-con-nrf52832-4, ./transport_plugins/native_ble, ./iotile_ext_cloud, requests-mock, tornado>=4.5.3,<5.0.0, pycryptodome, configparser
```